spec/spec_helper.rb

```diff
@@ -15,6 +15,7 @@
 
   config.around do |example|
     ensure_fixtures("spec", "fixtures", "schemas") do
+      JsonMatchers::Matcher.document_store = nil
```

Comment:
Why is it necessary to throw away our cache in each of the project's test suites?

Could you elaborate more on those details in the commit message?

Reply:
It clears shared state (cache) between test cases